extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/URIExtractionNamespaceFunctionFactory.java

```diff
@@ -151,9 +151,23 @@ public String call()
             versionRegex = null;
           }
         } else {
-          final Path filePath = Paths.get(extractionNamespace.getUri());
+          final URI rawURI = extractionNamespace.getUri();
+          final Path filePath = Paths.get(rawURI.getPath());
```

Comment:
I'm mostly asking those things because `Paths.get(String)` is going to parse the path as if it were on the local filesystem, which I'm not sure will always work? So if it turns out we don't actually _need_ to do that, it could be simpler to just not do it.